ceph-iscsi.spec

```diff
@@ -68,7 +68,7 @@ Requires:       python3-configshell
 %endif
 %endif
 
-%if 0%{?rhel} < 8
+%if 0%{?rhel} && 0%{?rhel} < 8
```

Comment:
Maybe just change these to ```0%{?rhel} == 7``` to avoid the need to wrap the extra conditionals?

Reply:
If you're sure ceph-iscsi will never be built for 6 and below, then :+1:

Reply:
100% sure -- not supported.